mobileapps/views.py

```diff
@@ -157,13 +157,18 @@ def get_queryset(self):
         queryset = MobileApp.objects.all()
         app_name = self.request.query_params.get('app_name', None)
         organization_name = self.request.query_params.get('organization_name', None)
+        organization_ids = self.request.query_params.get('organization_ids', None)
```

Comment:
i think we should use this helper `get_ids_from_list_param` from api-integration here.